ts/fusion-frontend/package.json

```
@@ -1,11 +1,11 @@
 {
   "name": "@vaadin/fusion-frontend",
   "version": "0.0.15",
-  "description": "Vaadin frontend utils used by Fusion",
+  "description": "Fusion frontend utils",
   "main": "index.js",
   "module": "index.js",
   "type": "module",
-  "repository": "https://github.com/vaadin/fusion/frontend/packages/fusion-frontend",
+  "repository": "https://github.com/vaadin/fusion/ts/fusion-frontend",
```

This is not a valid way to define a repository with subdirectory for the package. See https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

It should be like this (if using the `ts/fusion-frontend` dir):
```suggestion
  "repository": {
    "type" : "git",
    "url" : "https://github.com/vaadin/fusion.git",
    "directory": "ts/fusion-frontend"
  },
```